### DISCUSSION TITLE
How do I rename my servers in the network

### DISCUSSION THREAD

**Discussion by Charlie-Ma-1729** [plan-player-analytics/Plan#3515] [Q&A]
![圖片](https://github.com/plan-player-analytics/Plan/assets/61151352/dafc3441-2bac-4235-8c5e-9c07e6c59094)
so I have a network on but i don't know how to rename the servers.
I just don't want them to be Server 2, Server 3, Server 4...
I want to rename them so that I could tell which one it which better

**Comment by AuroraLS3** [comment]
ServerName option in the config
